src/common/Messages.h

```diff
@@ -24,6 +24,8 @@ enum MessageOffset : unsigned int { TYPE = 0, INST = 1, BODY = 2 };
 
 enum NumberSign : unsigned char { POSITIVE = 0x00, NEGATIVE = 0x01 };
 
+enum BoolSign : unsigned char { ISTRUE = 0x00, ISFALSE = 0x02 };
```

Comment:
Why need of 00 and 02. not 00 and 01

Reply:
After implemented the GetHash func in Messenger will remove this

Reply:
fixed